apps/portals/cancercomplexity/src/pages/CCKPSearchPage.tsx

```diff
@@ -41,7 +41,7 @@ export const searchPageTabs: PortalSearchTabConfig[] = [
     title: 'Educational Resources',
     path: 'EducationalResources',
   },
-]
+] satisfies PortalSearchTabConfig[]
```

Comment:
Sorry, this won't do anything unless you
1. remove `: PortalSearchTabConfig[]` above on L19
2. Here, use `as const` (i.e. `] as const satisfies PortalSearchTabConfig[]`)

But if you do that, then the `count` property cannot be dynamically changed.

So if you want to get this type-safety, then you would need to separate the count into a different object (maybe a `Map<(typeof searchPageTabs)[number]['title'], number>`?). Up to you if you want to do that work or just remove the `satisfies` here.

```suggestion
]
```